post_proc.py

```diff
@@ -1,8 +1,33 @@
-# Pra remover os espaços em branco no início, lstrip
-
 in_file = open("diario-anadia-2022-08-29-extraido.txt", "r")
-out_text = in_file.read().lstrip()
-out_file = open("diario-anadia-2022-08-29-proc.txt", "w")
-out_file.write(out_text)
-out_file.close()
-    
+# Mudei o nome da variável out_text para in_text, pois ela não será mais usada como texto de saída.
+in_text = in_file.read().lstrip()
+in_text = in_text.splitlines()
+ama_header_words = in_text[0].split()
+ama_header_id = ama_header_words[0]
+
+
+def generate_text():
+    final_text = ""
+    for linha in range(len(in_text)):
+        final_text += in_text[linha] + "\n"
+    return final_text
+# alterar as outras ocorrências do header do ama
+
+
+def delete_header():
+    for linha in range(len(in_text)):
+        if linha == 0:
+            continue
+        if in_text[linha].startswith(ama_header_id):
+            in_text[linha] = in_text[linha].replace(in_text[linha], "")
```

Comment:
Não seria a mesma coisa?

```suggestion
            in_text[linha] = ""
```